### DISCUSSION TITLE
skywalking9.0.0 kafka

### DISCUSSION THREAD

**Discussion by BritneyStars** [apache/skywalking#8952] [Q&A]
When skywalking is updated 9.0.0, agent is 8.7.0 , Lots of log infos about kafka in the skywalking oap-server background ,As follows：
[Consumer clientId=consumer-skywalking-1, groupId=consumer-skywalking] Attempt to heartbeat failed since group is rebalancing
[Consumer clientId=consumer-skywalking-consumer-1, groupId=consumer] Member consumer-skywalking-consumer-1-1-7d64e140-f0e3-49d2-8230-2621ba1d2061 sending LeaveGroup request to coordinator xxxx:9092 (id: 2147483643 rack: null) due to consumer poll timeout has expired. This means the time between subsequent calls to poll() was longer than the configured max.poll.interval.ms, which typically implies that the poll loop is spending too much time processing messages. You can address this either by increasing max.poll.interval.ms or by reducing the maximum size of batches returned in poll() with max.poll.records. 
  the kafka does not consumer the topic . but when skywalking go back to 8.9.1 , the logs about kafka does not have ,and the topic can be consumed normal.

**Comment by wu-sheng** [comment]
I am not aware of Kafka consumer changes in 9.0. You could check change logs, or log commits.

**Reply by BritneyStars** [reply]
There is nothing changes about kafka consumer inskywalking9.0.0

**Reply by BritneyStars** [reply]
kafka 's version is 0.11.0 . the version  of kafka has influence？
